src/common/detector/Detector.h

```diff
@@ -166,4 +176,8 @@ class Detector {
   std::atomic_bool runThreads{true};
   uint32_t RuntimeStatusMask{0};
   bool CalibrationMode{false};
+
+  // Common to all EFUs
+  EV44Serializer *Serializer;
```

Comment:
So have an EV44 Serializer is not common to all EFU's. The CBM has a vector of EV44 and DA00 for example. However I planning to create common interface btw. the two and I can imagine in the future to have one common vector of serializer's.

Also this should be not in public but protected or private. It better to have a controlled function to get this data then a read/write pointer.